src/routes/products/+page.svelte

```diff
@@ -13,67 +13,88 @@
       brand: 'Forzudo',
       name: 'Rojo',
       image: forzudoRojo,
+      region: 'Bierzo, Leon, Nordvest',
+      sizeAndDegrees: '100 cl. / 15%',
     },
-
     {
       brand: 'Forzudo',
       name: 'Blanco',
       image: forzudoBlanco,
+      region: 'Bierzo, Leon, Nordvest',
+      sizeAndDegrees: '100 cl. / 15%',
     },
-
     {
       brand: 'Sardino',
       name: 'Rojo',
       image: sardinoRojo,
+      region: 'Galicien, Nordvestkysten',
+      sizeAndDegrees: '75 cl. / 15%',
     },
     {
       brand: 'Sardino',
       name: 'Blanco',
       image: sardinoBlanco,
+      region: 'Galicien, Nordvestkysten',
+      sizeAndDegrees: '75 cl. / 15%',
     },
-
     {
       brand: 'Carmeleta',
       name: 'Orange ',
       image: carmeletaOrange,
+      region: 'L’Alquería de la Comtessa, Valencia',
+      sizeAndDegrees: '75 cl. / 15%',
     },
     {
       brand: 'Carmeleta',
       name: 'Blanco',
       image: carmeletaBlanco,
+      region: 'L’Alquería de la Comtessa, Valencia',
+      sizeAndDegrees: '75 cl. / 15%',
     },
-
     {
       brand: 'Tabira',
-      name: '-',
+      name: '',
       image: tabira,
+      region: 'Leon, Nordvest',
+      sizeAndDegrees: '100 cl. / 15%',
     },
   ]
 </script>
 
-<section>
-  <div class="container">
-    {#each vermouths as { image, name, brand }, index}
-      <Product {image} {name} {brand} {index} />
-    {/each}
-  </div>
+<section class="container">
+  {#each vermouths as { image, name, brand, region, sizeAndDegrees }, index}
+    <div class="wrap">
+      <Product {image} {name} {sizeAndDegrees} {brand} {region} {index} />
```

Comment:
seems a bit stupid to unwrap all these props here - you could just pass it the entire object and destructure in component...

Reply:
That has already changed in the PR when I unify I am going to have many things that do not add up